src/sdg/producten/models/product.py

```diff
@@ -206,6 +206,11 @@ class status(DjangoChoices):
         help_text=_("Heeft dit product kosten?"),
         default=False,
     )
+    api_zichtbaarheid = models.BooleanField(
```

Comment:
It should be the other way around. Can you rename this as suggested in the ticket:

> boolean field Product.hidden. Default is False.

You can name it `is_hidden_in_api` if you want, but "zichtbaarheid" field name suggests "visible" which is the opposite of hidden. Default should also be False.